apps/server/src/modules/tldraw/config.ts

```diff
@@ -17,14 +17,15 @@ export interface TldrawConfig {
 	API_HOST: number;
 	TLDRAW_MAX_DOCUMENT_SIZE: number;
 	TLDRAW_FINALIZE_DELAY: number;
+	PERFORMANCE_MEASURE_ENABLED: boolean;
 }
 
 export const TLDRAW_DB_URL: string = Configuration.get('TLDRAW_DB_URL') as string;
 export const TLDRAW_SOCKET_PORT = Configuration.get('TLDRAW__SOCKET_PORT') as number;
 
 const tldrawConfig = {
 	TLDRAW_DB_URL,
-	NEST_LOG_LEVEL: Configuration.get('NEST_LOG_LEVEL') as string,
+	NEST_LOG_LEVEL: Configuration.get('TLDRAW__LOG_LEVEL') as string,
```

Comment:
don't do that. you can set a different log level for tldraw anyway without affecting other services

Reply:
Ehrlich gesagt wüsste ich nicht wie.